src/main/resources/tech/aroma/sql/ddl/activity_ddl.sql

```diff
@@ -0,0 +1,21 @@
+------------------------------------------------------------------------------
+-- TABLES TO STORE ACTIVITY THAT HAPPENS WITHIN AROMA
```

Comment:
It's just one table here bro.
![bitmoji](https://render.bitstrips.com/v2/cpanel/8942765-288439948_4-s1-v1.png?transparent=1&palette=1&width=246)